include/psen_scan_v2/raw_processing.h

```diff
@@ -77,6 +77,18 @@ inline void readArray(std::istringstream& is,
   });
 }
 
+template <typename T>
+inline T serialize(std::ostringstream& os)
```

Comment:
Not clear what this function does. Given the name I would assume you can pass it a ostringstream and it returns an arbitrary type...

Reply:
renamed to toArray